content/news/2021-01-08-mixxx-to-the-first-time-contributor.md

```diff
@@ -0,0 +1,110 @@
+title: Mixxx to the first time contributor
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2021-01-08 17:38:00
+summary: Before the Outreachy internship program, I had never contributed to open-source software. I didn’t know what the term "open source" meant or how different it was from other software. Looking back, it's hard to believe how clueless I was four months ago and then compare to how far I've come. I think it is safe to say that I have learned a lot about different open source software, Mixxx in particular, and the different ways that I can contribute. I feel very fortunate to have been offered the Outreachy internship, and now I will share everything that I know about contributing to the Mixxx open source projects - from the perspective of a first time contributor :)
+
+![Mixxx logo]({static}/images/news/mixxx_logo.png)
+
+Mixxx is a free/open source DJ performance software for Windows/Mac/and Linux. It was first released in 2003 as a graduate student project, and was picked up by the community and greatly enhanced. All contributors, from programmers to skin designers to translators are unpaid volunteers, hence it is entirely community driven.
+
+Mixxx is a feature-rich DJ mixing application that supports many MIDI and HID DJ controllers.
+It supports effects, harmonic mixing, beat matching with thousands of users all over the world.
+It integrates the tools DJs need to perform creative live mixes with digital music files.
+Mixxx has an unusually broad community for an open-source project, encompassing performing musicians, C++ programmers and addicts, amateur DJs, Internet radio broadcasters and even casual users.
+
+Besides being a fully-featured DJing application, the main problem that Mixxx is trying to solve is that a lot of DJing applications are either expensive or limited (for the free features).
+Mixxx exists for people who want to try DJing or do not have the money to buy equipment or the latest fastest computers.
```

Comment:
There are also other reasons, e.g. most commercial competitors won't work on Linux. In contrast to those, it's also the only DJ software that you can fully customize to your needs, because you have full access to the source code.

I'm not sure if adding these features here makes sense in this context, but maybe we can rephrase the sentence above, e.g.:

> Since Mixxx is completely free and also runs on older hardware, it's popular among people who want to try DJing or do not have the money to buy equipment or the latest fastest computers.

or something like that

Reply:
> There are also other reasons, e.g. most commercial competitors won't work on Linux. In contrast to those, it's also the only DJ software that you can fully customize to your needs, because you have full access to the source code.

Thank you for this. The reasons why people choose to use Mixxx and what sets it apart from other DJing software are the foundation of this article. I really would love to build more on it. 
So in addition to these two points, maybe i can include a brief explanation of how Mixxx can be customised to people's needs (or what can be customised).

Is there a reason most commercial competitors won't work on Linux

> I'm not sure if adding these features here makes sense in this context, but maybe we can rephrase the sentence above, e.g.:
> 
> > Since Mixxx is completely free and also runs on older hardware, it's popular among people who want to try DJing or do not have the money to buy equipment or the latest fastest computers.
> 
> or something like that

This sounds better. I will use this. :)

Reply:
> Is there a major reason most commercial competitors won't work on Linux?

Linux has a low desktop marketshare, so it's probably not profitable enough for them.